### ISSUE TITLE
Usability: NVDA announces irrelavant information after opening the Git output pane.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17260]
VS Code Version: 1.6.1
OS Version:1607
NVDA version:-2016.2.1

**Steps to Reproduce:**
1.Open Visual studio code editor.
2. Open any file and using "Ctrl+1" key  navigate to the first vs code editor.
3. Using F1 key navigate to the palette, type "git branch" and "refactor-to-promise" as the branch name
4. Make changes to the code by refactoring, adding/deleting new lines
5. Compare the changes between the HEAD and current file by navigating to View Bar ---> Git ; Navigate to changes and hit ENTER to activate "Diff View" 
6. Stage and commit the changes under View Bar ---> Git  ; Navigate to the "Commit Staged"  button and hit ENTER to make a commit
7. Undo the commit by navigating to the toggle button (same row as commit staged), hit ENTER and use down ARROW to choose "Undo Last Commit" option 
8. Navigate to "More Button" in Git Side Bar and Hit Enter.
9. Navigate to "Show Git Output" and Hit Enter and observe NVDA announcement

**Expected result:**
NVDA should announce Proper Information regarding the output of the Git after opening the output pane.

**Actual Result:**
NVDA announces irrelevant information after opening the output pane
1. NVDA announces as "Pane unavailable; index.html (HEAD)<->index.html - yesho - Visual studio code document; note; Git actions toolbar; Pane unavailable; index.html (HEAD)<->index.html - yesho - Visual studio code document; note; Git actions toolbar;"

**User Impact:**
Screen Reader user gets confused if NVDA announces irrelevant information after opening the output pane

Attachments:
![nvda announces irrelavant information after opening the output pane](https://cloud.githubusercontent.com/assets/22956130/21214979/28b9d452-c2c4-11e6-8727-0b6664de1f7a.jpg)

**Comment by isidorn** [comment]
This works perfectly for me.
As I am navigating through the git output NVDA nicely reads out every line.

Closing as can not reproduce
@WiproPSAccTest please let me know if you still see this with latest VSCode insiders and latest NVDA. Thanks
